portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/Publications.testcase

```diff
@@ -567,6 +567,46 @@ definition {
 		MenuItem.viewNotPresent(menuItem = "Delete");
 	}
 
+	test CanPublishAfterDeleteOnPublicationAndProduction {
+		task ("Go to site to disable recycle bin and create 2 Pages") {
+			Site.openSiteSettingsAdmin(siteURLKey = "guest");
+
+			RecycleBin.disableRecycleBinViaSiteSettings();
+
+			for (var count : list "1,2") {
+				JSONLayout.addPublicLayout(
+					groupName = "Guest",
+					layoutName = "Page ${count}");
+			}
+		}
+
+		task ("Create Publication and delete both pages") {
+			JSONPublications.addPublication(publicationName = "P1");
+
+			JSONPublications.selectPublication(publicationName = "P1");
+
+			Refresh();
+
+			for (var count : list "1,2") {
+				JSONLayout.deletePublicLayout(
+					groupName = "Guest",
+					layoutName = "Page ${count}");
+			}
+		}
+
+		task ("Go to production mode, delete Page 1 and Publish Publication") {
+			Publications.workOnProduction();
```

Comment:
You'd better add a step to `PublicationsNavigator.openPublicationsAdmin();` in case you are not on the publications admin page.